pkgs/development/ocaml-modules/llvm/default.nix

```diff
@@ -12,12 +12,8 @@ stdenv.mkDerivation {
   buildInputs = [ python ocaml findlib ctypes ];
   propagatedBuildInputs = [ libllvm ];
 
-  patches = [ (fetchpatch {
```

Comment:
You may remove `fetchpatch` from the package arguments.

Reply:
done